specs/Standards/Storage.md

```diff
@@ -0,0 +1,206 @@
+# Storage Standard ([NEP-145](https://github.com/near/NEPs/discussions/145))
+
+Version `1.0.0`
+
+## Summary
+
+This standard handles state storage when accounts add and remove data stored in the state of a contract. It allows accounts to:
+
+1. Check an account's storage "balance."
+2. Determine the minimum storage needed to add account information such that the account can interact as expected in a contract.
+3. Add storage for an account; either their own or another account.
+4. Withdraw storage by removing associated account data from the contract.
```

Comment:
Updated to 

> Withdraw some or all of a storage deposit by removing associated account data from the contract and then making a call to remove unused deposit.

Reply:
I've also updated the "Future Possibilities" section with:

> - This standard avoids specifying how to close/unregister an account.
> - Instead of having developers use constants for the storage price per byte, perhaps this can be derived and instructions can be modified in this standard regarding how to calculate the value returned for `storage_minimum_balance`.
> - Ideally, contracts will update `available_balance` for all accounts every time the NEAR blockchain's configured storage-cost-per-byte is reduced.